.github/workflows/check-catalyst.yaml

```
@@ -43,7 +43,7 @@ jobs:
       - name: Checkout Catalyst repo
         uses: actions/checkout@v4
 
-      - name: Set up Python # Ensure python3.10 is used
+      - name: Set up Python # Ensure python3.11 is used
```

Maybe instead of updating this comment everytime we could change it to 
```suggestion
      - name: Set up Python # Ensure the "primary" python version is used
```